package.json

```diff
@@ -45,9 +45,9 @@
     "estraverse": "^4.2.0",
     "esutils": "^2.0.2",
     "file-entry-cache": "^1.1.1",
-    "glob": "^7.0.3",
+    "glob2": "^7.0.3",
```

Comment:
I am little scared to make this change because just for one functionality if we make a switch, we will loose out on future bug fixes and enhancements on the more widely used library.